dll/cpl/sysdm/lang/es-ES.rc

```diff
@@ -348,13 +348,13 @@ BEGIN
     IDS_USERPROFILE_STATUS "Estado"
     IDS_USERPROFILE_MODIFIED "Modificado"
     IDS_USERPROFILE_LOCAL "Local"
-    IDS_USERPROFILE_ROAMING "Roaming"
-    IDS_USERPROFILE_MANDATORY "Mandatory"
-    IDS_USERPROFILE_CONFIRM_DELETE "Are you sure you want to delete %s's profile?"
-    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirm Delete"
-    IDS_USERPROFILE_ACCOUNT_UNKNOWN "Account Unknown"
-    IDS_USERPROFILE_ACCOUNT_DELETED "Account Deleted"
-    IDS_USERPROFILE_TYPE_TEXT "When %s logs onto this computer, should the operating system use the roaming profile or just the locally cached copy of the roaming profile."
+    IDS_USERPROFILE_ROAMING "Móvil"
+    IDS_USERPROFILE_MANDATORY "Obligatorio"
+    IDS_USERPROFILE_CONFIRM_DELETE "¿Está seguro de que quiere borrar el perfil %s'?"
+    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirme borrado"
```

Comment:
I would preffer "Confirmar eliminación"

Reply:
So do I, or maybe other formula, like "confirmar borrado", because coherence of the same style.

Reply:
I think "borrado" is more emphatic to the end-user than "eliminación". He will probably pay more attention.